src/app/(root)/terms/[id]/page.tsx

```
@@ -0,0 +1,24 @@
+  };
+}
+
+export default function Page({ params }: { params: { id: string } }) {
+  const term = getTermById(params.id);
+  if (!term) {
+    return <div className="prose max-w-none"><h1>Term not found</h1></div>;
+  }
+  return (
+    <article className="prose max-w-none">
+      <h1 className="text-3xl font-semibold">{term.term}</h1>
+      <div className="mt-4" dangerouslySetInnerHTML={{ __html: term.definition }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)